target/linux/ath79/dts/qca9558_allnet_all-wap02860ac.dts

```diff
@@ -95,88 +56,10 @@
 	};
 };
 
-&pcie0 {
-	status = "okay";
-
-	wifi@0,0 {
-		compatible = "qcom,ath10k";
-		reg = <0 0 0 0 0>;
-	};
-};
-
-&pll {
-	clocks = <&extosc>;
-};
-
-&spi {
-	status = "okay";
-
-	flash@0 {
-		compatible = "jedec,spi-nor";
-		reg = <0>;
-		spi-max-frequency = <25000000>;
-
-		partitions {
-			compatible = "fixed-partitions";
-			#address-cells = <1>;
-			#size-cells = <1>;
-
-			partition@0 {
-				label = "u-boot";
-				reg = <0x000000 0x040000>;
-				read-only;
-			};
-
-			partition@40000 {
-				label = "u-boot-env";
-				reg = <0x040000 0x010000>;
-			};
-
-			partition@50000 {
-				label = "custom";
-				reg = <0x050000 0x050000>;
-				read-only;
-			};
-
-			partition@a0000 {
-				label = "loader";
-				reg = <0x0a0000 0x010000>;
-			};
-
-			fwconcat1: partition@b0000 {
-				label = "fwconcat1";
-				reg = <0x0b0000 0x180000>;
-			};
-
-			fwconcat0: partition@230000 {
-				label = "fwconcat0";
-				reg = <0x230000 0xb40000>;
-			};
-
-			partition@d70000 {
-				label = "failsafe";
-				reg = <0xd70000 0x280000>;
-				read-only;
-			};
-
-			art: partition@ff0000 {
-				label = "art";
-				reg = <0xff0000 0x010000>;
-				read-only;
-			};
-		};
-	};
-};
-
-&uart {
-	status = "okay";
-};
-
 &wmac {
 	status = "okay";
 
 	mtd-cal-data = <&art 0x1000>;
 	mtd-mac-address = <&art 0x0>;
 	mtd-mac-address-increment = <1>;
-	qca-no-eeprom;
```

Comment:
Why is this deleted?

Reply:
1. spelled wrong so there is no effect
2. mtd-cal-data takes care of this

Reply:
Ack, because it was mistyped 'mtd-cal-data' actually worked.